Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -56,20 +56,46 @@
 #include <wtf/text/AtomStringHash.h>
 #include <wtf/text/StringToIntegerConversion.h>
 
+static void videoDecoderLimitsDefaults(unsigned& mediaMaxWidth, unsigned& mediaMaxHeight, unsigned& mediaMaxFrameRate)
+{
+#ifdef VIDEO_DECODING_LIMIT
+    // VIDEO_DECODING_LIMIT should be in format: WIDTHxHEIGHT@FRAMERATE
+    String videoDecodingLimit(String::fromUTF8(VIDEO_DECODING_LIMIT));
+
+    if (videoDecodingLimit.isEmpty())
+        return;
+
+    Vector<String> entries;
+
+    // extract frame rate part from the VIDEO_DECODING_LIMIT: WIDTHxHEIGHT@FRAMERATE
```

Comment:
I don't sympathize a lot with this WebKit style rule, but... comments must be formal sentences, starting with capital leter and finishing with a dot (.).